example/fcn-xs/README.md

```diff
@@ -17,32 +18,36 @@ We have trained a simple fcn-xs model, the hyper-parameters are below:
 
 The training dataset size is only 2027, and the validation dataset size is 462.  
 
-## How to train fcn-xs in mxnet
-#### Getting Started
+## Training the model
+
+### Step 1: setup pre-requisites
 
 - Install python package `Pillow` (required by `image_segment.py`).
 ```shell
-[sudo] pip install Pillow
+pip install Pillow
```

Comment:
Will be good to give -U options for pip..

Reply:
Good point, updated to --upgrade for clarity